dask/array/tests/test_routines.py

```diff
@@ -221,6 +221,36 @@ def test_flip(funcname, kwargs, shape):
         assert_eq(np_r, da_r)
 
 
+@pytest.mark.parametrize(
+    "kwargs",
+    [{}, {"axes": (1, 0)}, {"axes": (2, 3)}, {"axes": (0, 1, 2)}, {"axes": (1, 1)}],
+)
+@pytest.mark.parametrize("shape", [tuple(), (4,), (4, 6), (4, 6, 8), (4, 6, 8, 10)])
+def test_rot90(kwargs, shape):
+    axes = kwargs.get("axes", (0, 1))
+    np_a = np.random.random(shape)
+    da_a = da.from_array(np_a, chunks=1)
+
+    np_func = getattr(np, "rot90")
+    da_func = getattr(da, "rot90")
+
+    try:
+        for axis in axes[:2]:
+            range(np_a.ndim)[axis]
+    except IndexError:
+        with pytest.raises(ValueError):
+            da_func(da_a, **kwargs)
+    else:
+        if len(axes) != 2 or axes[0] == axes[1]:
+            with pytest.raises(ValueError):
+                da_func(da_a, **kwargs)
+        else:
+            for k in range(4):
```

Comment:
Should we try something larger than 4 to test the modulus? What about negative values?

Reply:
I thought the same thing but wasn't sure. The numpy unit tests have the following:

https://github.com/numpy/numpy/blob/cb557b79fa0ce467c881830f8e8e042c484ccfaa/numpy/lib/tests/test_function_base.py#L68-L75

```python
        for k in range(-3, 13, 4):
            assert_equal(rot90(a, k=k), b1)
        for k in range(-2, 13, 4):
            assert_equal(rot90(a, k=k), b2)
        for k in range(-1, 13, 4):
            assert_equal(rot90(a, k=k), b3)
        for k in range(0, 13, 4):
            assert_equal(rot90(a, k=k), b4)
```

to test the modulus is working. Since we check against numpy, I think `k in range(-3, 13)` or `k in range(-3, 9)` should probably be sufficient. Thoughts?

Reply:
Sure I think `range(-3, 9)` should be fine. Out of curiosity how long does this test take to run before/after that change?

Reply:
Not sure if there is a more specific timing mechanism you'd prefer:

`for k in range(4)`

```
> time py.test dask/array/tests/test_routines.py::test_rot90
# ...
py.test dask/array/tests/test_routines.py::test_rot90  4.11s user 0.37s system 125% cpu 3.574 total
```

`for k in range(-3, 9)`

```
> time py.test dask/array/tests/test_routines.py::test_rot90
# ...
py.test dask/array/tests/test_routines.py::test_rot90  9.18s user 0.44s system 110% cpu 8.720 total
```

Reply:
That seems fine. Let's go ahead and add them then

Reply:
done!